src/field.cpp

```diff
@@ -332,8 +332,21 @@ void game::init_fields()
             {_("hazy cloud"),_("fungicidal gas"),_("thick fungicidal gas")}, '8', 8,
             {c_white, c_ltgray, c_dkgray}, {true, true, false}, {false, true, true}, MINUTES(90),
             {0,0,0}
-        }
+        },
 
+		{
+			"fd_ink",
+			{ _("ink splatter"), _("ink stain"), _("puddle of ink") }, '%', 0,
+			{ c_ltgray, c_ltgray, c_dkgray }, { true, true, true }, { false, false, false }, HOURS(24),
+			{ 0,0,0 }
+		},
+
+		{
+			"fd_ink_cloud",
```

Comment:
There should not be any cases were fd_ink_cloud would spawn outside swimable terrain. But I could make it turn into fd_ink the on dry terain.